templates/base.html

```diff
@@ -1,5 +1,6 @@
 <!doctype html>
 <meta charset=utf-8>
+<meta name=viewport content="width=device-width, user-scalable=no">
```

Comment:
Latest recommendations are more complete: https://codepen.io/tigt/post/meta-viewport-for-2015